src/components/SearchBar/SearchBar.tsx

```diff
@@ -50,6 +44,12 @@ export const SearchBar = (props: ISearchBarProps): ReactElement => {
     }
   };
 
+  // on clear button press, clear query and focus
+  const handleClear = () => {
+    updateQuery('');
```

Comment:
Just want to confirm: the `query' in store is updated whenever the search input is updated. The query in the store is not the current search, but what will be searched when user click enter?

Reply:
that's correct.  I do see though how that might not be desirable depending on the use case.  I would say that this isn't a show stopper since it's just concern with store structure, which is easy enough to alter/add right now if necessary